src/main/scala/se/nimsa/dcm4che/streams/DicomPartFlow.scala

```diff
@@ -52,13 +51,13 @@ class DicomPartFlow(chunkSize: Int = 8192, stopTag: Option[Int] = None, inflate:
       val explicitVR: Boolean
     }
 
-    case class DatasetHeaderState(bigEndian: Boolean, explicitVR: Boolean) extends HeaderState
+    case class DatasetHeaderState(itemIndex: Int, bigEndian: Boolean, explicitVR: Boolean) extends HeaderState
```

Comment:
We now keep track of the index of the current item when streaming through sequences and fragments.